app/Http/Controllers/Api/WordController.php

```diff
@@ -0,0 +1,233 @@
+                            "name" => $item,
+                            "is_correct_answer" => "1",
+                        ];
+                    } else {
+                        $choice = [
+                            "word_id" => $data[0],
+                            "name" => $item,
+                        ];
+                    }
+                    // Store the array data choice
+                    Choice::create($choice);
+                });
+                return response()->json([
+                    "error" => false,
+                    "message" => "Successfull Saved",
+                ]);
+            } catch (QueryException $e) {
+                return response()->json([
+                    "error" => true,
+                    "message" => "Error in saving data. Try again.",
+                    "error_message" => $e,
+                ]);
+            }
+        }
+        return response()->json([
+            "error" => true,
+            "message" => "Invalid Fields",
+        ]);
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function show($category_id)
+    {
+        try {
+            $word = Word::where("category_id", $category_id)
+                ->with("choices")
+                ->paginate();
+
+            return response()->json([
+                "error" => false,
+                "message" => "Succesfully queried data",
+                "data" => $word,
+            ]);
+        } catch (QueryException $e) {
+            return response()->json([
+                "error" => false,
+                "message" => "Succesfully queried data",
+                "errorMessage" => $e,
+            ]);
+        }
+    }
+
+    /**
+     * Show the form for editing the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function edit($id)
+    {
+        //
+    }
+
+    /**
+     * Update the specified resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function update(Word $word, WordUpdateRequest $request)
+    {
+        if ($request->validated()) {
+            try {
+                // Update word.name
+                $word->update($request->only("name"));
+                // Update choices.name first row
+                $answer = $request->correct_answer;
+                $word
+                    ->choices()
+                    ->first()
+                    ->update([
+                        "name" => $request->choices[0],
+                        "is_correct_answer" => $answer === "1" ? 1 : 0,
+                    ]);
+                // Update choices.name second row
+                $word
+                    ->choices()
+                    ->skip(1)
+                    ->first()
```

Comment:
I think you can access the second choice by treating it as an array...

```
->choices()[0] // first
->choices()[1] // second
.... so on
```

Reply:
I tried to used that approach ma'am. I did `->choices()[0]->update()` and `->choices()[0]->first()->update()` but it gives an error "Cannot use object of type Illuminate\Database\Eloquent\Relations\HasMany as array"

Reply:
@michaelsilverio Ay sorry, i typed it wrong... dapat `->choices[0]` or `->choices()->get()[0]`